content/en/docs/score specification/score-spec-reference.md

```diff
@@ -287,17 +287,13 @@ resources: # optional
       param-name: value
 ```
 
-`resources`: the resource dependencies needed by the workload.
-
-`resource-name`: a required property that specifies the resource name.
-
-`type`: the resource type. This should be a type supported by the Score implementations being used.
-`class`: an optional specialisation of the resource type.
-`id`: an optional external resource identifier. When two resources share the same type, class, and id, they are considered the same resource when used across related Workloads.
-`params`: an optional map of parameters that may configure this resource. Params support both metadata and resource placeholders local to this Workload.
-
-`metadata`: an optional property that specifies additional resource metadata.
-
+- `resources`: the resource dependencies needed by the workload.
```

Comment:
```suggestion
```

Reply:
To be consistent with the `service` and `containers` section we don't need to repeat this here, it's already mentioned above. Let's clean this up while we are here.